resources/web/wwi/Toolbar.js

```diff
@@ -830,6 +833,39 @@ export default class Toolbar {
     };
   }
 
+  #createEnableFollow() {
+    const followLi = document.createElement('li');
+    followLi.id = 'enable-follow';
+    this.settingsList.appendChild(followLi);
+
+    let label = document.createElement('span');
+    label.className = 'setting-span';
+    label.innerHTML = 'Viewpoint follow';
```

Comment:
To be more consistent with what we have in Webots / View menu:
```suggestion
    label.innerHTML = 'Follow Object';
```

I noticed a small inconsistency in the menu names where sometimes we use capital letter for all words and sometimes only for the first word, e.g. , Ambient Occlusion / Playback speed.

I believe we should always use Ambient Occlusion style as we do in Webots.